File: tools-starter-swagger/src/main/java/cn/gjing/tools/swagger/SwaggerService.java
package cn.gjing.tools.swagger;

import lombok.*;

/**
 * @author Gjing
 **/
@Getter
@Setter
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class SwaggerService {
    /**
     * 文档标签名
     */
    private String view;

    /**
     * 目标服务名或者文档地址
     */
    private String service;
}

File: tools-starter-swagger/src/main/java/cn/gjing/tools/swagger/SwaggerService.scala
package cn.gjing.tools.swagger

import scala.beans.BeanProperty

/**
 * @author Gjing
 **/
class SwaggerService {
  /**
   * Tab标签名
   */
  @BeanProperty
  var view: String = _

  /**
   * 目标服务path
   */
  @BeanProperty
  var service: String = _

  def this(view: String, service: String) {
    this
    this.view = view
    this.service = service
  }

}